include/numerics/petsc_solver_exception.h

```diff
@@ -22,7 +22,7 @@
 
 #ifdef LIBMESH_HAVE_PETSC
 
-#include "libmesh_exceptions.h"
```

Comment:
Wait, really?  How was this compiling before?

Reply:
Yeah, I have no clue... it should definitely have failed the standalone header check for one thing.